apiaccess.py

```diff
@@ -125,13 +145,18 @@ def name_evidence(evidence, naming):
 
 
 def mentions_to_evidence(mentions):
-    """Convert mentions (from /parse) to evidence structure as expected by the /diagnosis endpoint."""
-    return [{'id': m['id'], 'choice_id': m['choice_id'], 'initial': True} for m in mentions]
+    """Convert mentions (from /parse endpoint) to evidence structure as
+    expected by the /diagnosis endpoint.
+    """
+    return [{'id': m['id'], 'choice_id': m['choice_id'], 'initial': True}
+            for m in mentions]
 
 
 def question_answer_to_evidence(question_struct_item, observation_value):
-    """Return new evidence obtained via abswering the one item contained in a question with the given observation
-    value (status)."""
+    """Return new evidence obtained via abswering the one item contained in a
```

Comment:
I'd rather keep it as a list (even if it's bound to be one-element list) to maintain consistent naming — _evidence_ means a list of observations, not a single observation. This would be consistent with the above function, but also, with the naming convention used in the API.